client/lib/sections-utils/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External Dependencies
+ */
+import escapeRegExp from 'lodash/escapeRegExp';
+
+export function pathToRegExp( path ) {
```

Comment:
this is a duplicated function.
we could use the version in `server/bundler/utils`

Reply:
We can't import server code from client code, but I'll do the opposite